recipes/libwebsockets/all/conanfile.py

```diff
@@ -246,6 +246,11 @@ def validate(self):
             raise ConanInvalidConfiguration("Library hubbub not implemented (yet) in CCI")
             # TODO - Add hubbub package when available.
 
+        if self.options.with_ssl == "openssl":
+            no_zlib = self.dependencies["openssl"].options.get_safe("no_zlib")
+            if not no_zlib and self.options.with_zlib != "zlib":
+                raise ConanInvalidConfiguration(f"{self.name} needs option with_zlib=zlib, as its dependency openssl links with it")
```

Comment:
To discuss with the team what variant we want, either it showing
`libwebsockets/*:with_zlib=zlib` or `libwebsockets/x.y.z:with_zlib`, and what is more UX friendly